README.md

```diff
@@ -63,17 +61,40 @@ Dette feltet er med for eventtypen `done` for å sikre at man får med alle even
 
 
 ### Statusoppdatering
-TBA
+En sak eller søknad i et fagsystem har endret status, og dette ønskes å vises for sluttbruker.
+
+#### tidspunkt
+Et tidspunkt som noe skjedde, f.eks. da saksbehandlingen av en søknad var ferdig.
+
+#### fodselsnummer
+Fødselsnummeret til brukeren som eventet tilhører.
+
+#### grupperingsId
+Feltet grupperingsId brukes for å kunne samle alle eventer som hører til en sak, søknad eller et dokument. Dette er typisk en saksId, søknadsId eller dokumentId, men dere velger selv hvilken verdi dere putter der. Men det er viktig at det er samme verdi for alle eventer som skal gruppers samme. Typisk bruke av dette som vi ser for oss er oppbygging av tidslinjer.
```

Comment:
Det ser ut som at det en n i sammen, i slutten av den nest siste setningen.

Den siste setningen her flyter ikke så godt synes jeg, kunne vi skrevet den om litt? F.eks. "Typisk bruk for dette feltet vil være oppbygging av tidslinjer.".